### ISSUE TITLE
Scoverage plugin for jenkins shows 404 error file not found for some of the scala files but works on some

### ISSUE THREAD

**Issue by c3-yashgupta** [scoverage/sbt-scoverage#386]
Scoverage plugin for jenkins shows 404 error file not found for some of the scala files but works with rest. The pattern is kind of random For the files where I am not able to visually see the missing coverage(the html file is missing, dont know why) the corresponding Scoverage.xml file contains the coverage data for that particular file like missing branches etc. Has anybody faced this issue recently or know if it a known bug?

**Comment by ckipp01** [comment]
Hi @c3-yashgupta, thanks for the report. What version of the plugin are you using?

**Comment by c3-yashgupta** [comment]
> Hi @c3-yashgupta, thanks for the report. What version of the plugin are you using?

I am using `1.5.1`

**Comment by ckipp01** [comment]
Gotcha, any more context about when this is happening or patterns that you can spot will be helpful. To be honest I haven't really looked at the jenkins plugin at all, so I'll add that to the things to look into. It may be a bit as all effort at the moment is going towards getting Scala 3 support it.

**Comment by c3-yashgupta** [comment]
Yeah I tried going through the xml file as well to see if there is some difference in the row and root tags. But there is no distinguishing pattern that I could identify. Do you think updating to version `1.9.1` would help?

**Comment by ckipp01** [comment]
Possibly. There have been a fair amount of changes since then, especially in the actual scalac-scoverage-plugin. So I'm assuming bumping that will also bring in quite a bit newer of a plugin which may improve some of the reporting.
